_posts/2020-07-13-our-code-review-etiquette.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: Our Code Review Etiquette
+date: 2020-07-13 09:30:00
+categories: [best-practices]
+author: cleiviane
+---
+
+Code Review is one of the greatest tools we have as software developers to help us improve the quality of our code. It can be incredibly beneficial, but it can also be a source of pain, frustration, and overall, a waste of time instead of a time-saver.
+
+Because of that a while ago we wrote these code review tips that should be acknowledged and incorporated by everyone in our team and now we want to share them with you.
+
+<!--more-->
+
+Code Review it’s a way of sharing responsibilities and learn from each other. It’s where we can catch bugs and mistakes and it's also a way to enforce broadly agreed code standards between team members.
+
+But besides knowing what code review is for, we need to know what code review isn't for:
+
+- It is not the place for shooting at your teammates and pointing fingers at their mistakes.
```

Comment:
```suggestion
- It is not a place for shooting at your teammates and pointing fingers at their mistakes.
```
Hmmm, I'm not sure that "shooting" is quite the right word here, but I am also not quite sure what is. Maybe  one of these : "bullying", "giving your teammates a hard time" or "putting your teammates down".